File: app/src/debug/java/com/weathercock/android_cleanarchitecture/DebugCleanApplication.java
package com.weathercock.android_cleanarchitecture;

import com.facebook.stetho.Stetho;
import com.uphyca.stetho_realm.RealmInspectorModulesProvider;

/**
 * Application Class for Debug
 * <p>
 * Created by weath on 2016/06/05.
 */
public class DebugCleanApplication extends CleanApplication {

    @Override
    public void onCreate() {
        super.onCreate();

        Stetho.initialize(
                Stetho.newInitializerBuilder(this)
                        .enableDumpapp(Stetho.defaultDumperPluginsProvider(this))
                        .enableWebKitInspector(RealmInspectorModulesProvider.builder(this).build())
                        .build());
    }

}

File: app/src/debug/java/com/kazakago/cleanarchitecture/DebugCleanApplication.kt
package com.kazakago.cleanarchitecture

import com.facebook.stetho.Stetho

class DebugCleanApplication : CleanApplication() {

    override fun onCreate() {
        super.onCreate()
        initializeStetho()
    }

    private fun initializeStetho() {
        Stetho.initializeWithDefaults(this)
    }

}
